lib/cartopy/examples/utm_all_zones.py

```diff
@@ -0,0 +1,47 @@
+"""
+Displaying all 60 zones of the UTM projection
+---------------------------------------------
+
+This example displays all 60 zones of the Universal Transverse Mercator
+projection next to each other in a figure.
+
+First we create a figure with 60 subplots in one row.
+Next we set the projection of each axis in the figure to a specific UTM zone.
+Then we add coastlines, gridlines and the number of the zone.
+Finally we add a supertitle and display the figure.
+"""
+__tags__ = ['Lines and polygons']
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+
+
+def main():
+    # Create a list of integers from 1 - 60
+    zones = range(1, 61)
+
+    # Create a figure and array of 60 subplot Axes objects
+    fig, axarr = plt.subplots(nrows=1,
+                              ncols=len(zones),
+                              figsize=(18, 6))
+
+    # Loop through each zone in the list
+    for zone in zones:
+
+        # Change the Axes object to a GeoAxes object with a specific UTM zone projection
```

Comment:
@kaedonkers Could you make this line [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) compliant, by splitting it and ensuring that the line length is <80 characters.